### PR TITLE
fix(metabase-cronjob): allow custom command

### PR DESCRIPTION
**Pull request by revolunet**

### CODE REVIEW

**Comment by devthejo** [comment]
are you sure you want to provide an override by default (adding command: in values.yaml) ?
I'll suggest a change

File: plugins/contrib/charts/metabase/templates/matomo/cronjob.yaml [hunk 1]
```diff
@@ -31,10 +31,8 @@ spec:
               image: node:14-alpine
               securityContext:
                 allowPrivilegeEscalation: false
-              command:
-                - sh
-                - "-c"
-                - npx @socialgouv/matomo-postgres
+              command: 
+                {{- .Values.matomo.cronjob.command | toYaml | nindent 16 }}
```
Comment (devthejo) [inline plugins/contrib/charts/metabase/templates/matomo/cronjob.yaml:35]:
```suggestion
              {{- if .Values.matomo.cronjob.command }}
              command: 
                {{- .Values.matomo.cronjob.command | toYaml | nindent 16 }}
              {{- end }}
```

**Comment by revolunet** [comment]
> are you sure you want to provide an override by default (adding command: in values.yaml) ? I'll suggest a change

i think its good to have an overridable default ?

**Comment by devthejo** [comment]
> > are you sure you want to provide an override by default (adding command: in values.yaml) ? I'll suggest a change
> 
> i think its good to have an overridable default ?

It's an uncommon pattern to override the default command on an image that we master, but I didn't see that was not the case and that was a common node-14 image, so I understand better. OK

**Review by devthejo — APPROVED** [review]

**Pull request merged**